### PR TITLE
Bug fix 513

### PR DESCRIPTION
**Pull request by b0bkaT**
This Pull Request resolves all requirements of Issue #513 .

# Coordination

- [x] Pull Request is against correct branch
- [x] Pull Request is in, or reverted to, Draft status before Solutions Approval vote has passed
- [x] CI passes in UCO feature branch against `develop`
- [x] CI passes in UCO current `unstable` branch ([b22d045](https://github.com/ucoProject/UCO-Archive/commit/b22d0451d229f95fb2c5c6c81f2656540e37c117))
- [x] CI passes in CASE current `unstable` branch tracking UCO's `unstable` as submodule ([18fe007](https://github.com/casework/CASE-Examples/pull/124/commits/33e426251b03150e437aceda07a220516c13cdbc))
- [x] Impact on SHACL validation [reviewed](https://github.com/casework/CASE-Examples/pull/124/commits/33e426251b03150e437aceda07a220516c13cdbc) for CASE-Examples
- [x] Impact on SHACL validation remediated for CASE-Examples *(N/A)*
- [x] Impact on SHACL validation [reviewed](https://github.com/casework/casework.github.io/pull/231/commits/4fb6dde01647a9b7214625601b59b251a4f55324) for casework.github.io
- [x] Impact on SHACL validation [remediated](#TODO-commit) for casework.github.io *(N/A)*
- [x] Milestone linked
- [x] Solutions Approval vote logged on corresponding Issue (once logged, can be taken out of Draft PR status) *(N/A, bugfix)*

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
@DrSnowbird , could you please test out this PR and see if it resolves [Issue 513](https://github.com/ucoProject/UCO/issues/513)?  For some reason, I can't request you as a reviewer.

**Comment by DrSnowbird** [comment]
Alex,I will test it this afternoon - just so many piling-up tasks.v/r  Ray
    On Friday, January 13, 2023 at 03:13:16 PM EST, Alex Nelson ***@***.***> wrote:  
 
 


@DrSnowbird , could you please test out this PR and see if it resolves Issue 513? For some reason, I can't request you as a reviewer.

**Comment by DrSnowbird** [comment]
Hi Alex,
I just completed the tests and the previous SHACL errors in UCO Shapes are gone.
The fix Bug-Fix-153 is good as my review results. The fixed UCO is very good now (no error using my (TopBraid lib to validate).
V/R  Ray
    On Friday, January 13, 2023 at 03:30:52 PM EST, Ray Sheu ***@***.***> wrote:  
 
  Alex,I will test it this afternoon - just so many piling-up tasks.v/r  Ray
    On Friday, January 13, 2023 at 03:13:16 PM EST, Alex Nelson ***@***.***> wrote:  
 
 


@DrSnowbird , could you please test out this PR and see if it resolves Issue 513? For some reason, I can't request you as a reviewer.

**Comment by ajnelson-nist** [comment]
@DrSnowbird Wonderful, thank you!  We'll have this in the next release.

File: ontology/owl/owl.ttl [hunk 1]
```diff
@@ -226,7 +226,8 @@ uco-owl:ObjectProperty-shacl-constraints-shape
 			a sh:SPARQLConstraint ;
 			sh:message "An OWL Object Property must not permit a Literal value via SHACL consraints."@en ;
 			sh:select """
-				PREFIX sh: <http://www.w3.org/ns/shacl#>
+				PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+                PREFIX sh: <http://www.w3.org/ns/shacl#>
```
Comment (ajnelson-nist) [inline ontology/owl/owl.ttl:230]: Looks like tabs and single-spaces got confused.  I'll push an aligning patch.

**Review by ajnelson-nist — APPROVED** [review]
One bit of syntax style should be fixed, but otherwise this looks good.

File: ontology/owl/owl.ttl (hunk 1 above)
Comment (ajnelson-nist) [reply ontology/owl/owl.ttl]: Thanks, @b0bkaT , for the aligning patch!

**Pull request merged**